docs/vtex-io/Perks/best-practices/vtex-io-documentation-developing-custom-storefront-components.md

```diff
@@ -64,160 +66,160 @@ function Shelf({ title }: Props) {
 
-Before implementing your component, check which variables are available on browser using the [Render Runtime](https://github.com/vtex-apps/render-runtime#canusedom)'s `canUseDom` variable.
+Before implementing your component, check which variables are available on the browser using the [Render Runtime](https://github.com/vtex-apps/render-runtime#canusedom) `canUseDom` variable.
 
 ### Browser compatibility
 
-The VTEX IO platform automatically provides JavaScript polyfills responsible for implementing features on web browsers that are not able to.
+The VTEX IO platform automatically provides JavaScript polyfills for implementing features on web browsers that cannot do that.
 
-Although this practice ensures the good performance of your component in several user agents, you may work with NPM libraries that can not be implemented in old web browsers.
+Although this practice ensures the good performance of your component in several user agents, you may be working with NPM libraries that cannot be implemented in old web browsers.
 
-Check out which web browsers are more common among your audience and focus on them when developing your storefront component. Despite the fact that the Internet Explorer is falling into disuse, some stores are still IE11-oriented, for example.
+Check which web browsers are more common among your audience and focus on those when developing your storefront component. For example, some stores are still IE11-oriented, even though it is not as widely used anymore.
 
 ### Prop substitution
 
-The simplicity presented by boolean props, which toggle component behaviors between `true` and `false`, makes them well-employed in React projects.
+The simplicity of boolean props, with switch component behaviors `true` and `false`, makes them easy to use in React projects.
 
-However, they do not present great scalability considering that the component evolution may ask for a third (or more) behavior to be enabled.
+However, they do not offer great scalability when taking into account that the component evolution may require a third (or more) behavior to be enabled.
 
-Boolean props, therefore, demand creating brand new props whenever unprecedented use cases are introduced to your component.
+Boolean props, therefore, demand creating brand-new props whenever unprecedented use cases are introduced to your component.
 
-> ⚠️ Once the component is deployed, avoid managing its props (creating, removing, and/or updating them)! Managing props can be costly since it can directly impact the API's maintenance and therefore harm other developers working with it.
+> ⚠️ Once the component is deployed, avoid modifying its props (creating, removing, and/or updating them)! Modifying props can be costly because it can directly impact API maintenance and affect other developers working with it.
```

Comment:
```suggestion
> ⚠️ Once the component is deployed, avoid modifying its props (creating, removing, and/or updating them). Modifying props can be costly because it can directly impact API maintenance and affect other developers working with it.
```